start-gateway.sh

```diff
@@ -22,7 +22,8 @@ if [ -f .env-saved ]; then
     echo -e "Found saved environment from a previous run:\n"
     cat .env-saved
     echo
-    read -p  "Do you want to re-use the saved paramters? [y/N]: " REUSE_SAVED
+    read -p  "Do you want to re-use the saved paramters? [y]: " REUSE_SAVED
```

Comment:
Convention is to show the user valid options, with the default in caps
```suggestion
    read -p  "Do you want to re-use the saved paramters? [Y/n]: " REUSE_SAVED
```